README.md

```diff
@@ -12,6 +12,8 @@ SwiftGen is a suite of tools written in Swift to auto-generate Swift code (or an
 * [`enums` for your NSStoryboards and their Scenes](#nsstoryboard)
 * [`enums` for your Colors](#uicolor-and-nscolor).
 * [`enums` for your Fonts](#uifont-and-nsfont).
+* [`classes` for your models](#models)
+* [`any code` for your JSON specs](#json)
```

Comment:
This whole `any code` section seems extraneous to me. I think there should be one section "Parse your `json` files", and then one subsection in that about parsing models.